src/reanimated2/UpdateProps.ts

```diff
@@ -20,7 +28,7 @@ if (shouldBeUseWeb()) {
   updateProps = (_, updates, maybeViewRef, isAnimatedProps) => {
     'worklet';
     if (maybeViewRef) {
-      maybeViewRef.items.forEach((item, _) => {
+      maybeViewRef.items.forEach((item, __) => {
```

Comment:
Is there any way to avoid such change without doing `// eslint-disable-next-line`?  Like adding exception to eslint rule?

Reply:
Unfortunately no 😔